release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -474,3 +474,11 @@ request timeout. Even when a user exceeds the LDAP request timeout, the HTPasswd
 identity provider credentials should be used. The login process succeeds if the
 order of HTPasswd and LDAP identity provider usage is reversed.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1806620[*BZ#1806620*])
+
+* There is an issue with the Machine Config Operator (MCO) supporting Day 2
+proxy support, which describes when an existing non-proxied
+cluster is reconfigured to use a proxy. The MCO should apply newly configured
+proxy CA certificates in a ConfigMap to the {op-system} trust bundle; this is
+not working. As a workaround, you must manually add the proxy CA certificate to
+your trust bundle and then update the trust bundle.
```

Comment:
Does it make sense to provide more details for manually updating the trust bundle? For example:
````
  - create /etc/pki/ca-trust/source/anchors/my-root-ca.crt
  - update-ca-trust extract
  - drain node
  - systemctl reboot
```
I think the same can be accomplished through MCO, but I don't know enough about machine configs to be sure. Maybe @mrunalp can provide guidance.

Reply:
@danehans I've added the workaround details (similar to what you described above) in b947572f9340faae345085a912936fee00b0a0c9. Can you confirm?